docs/sketches.rst

```diff
@@ -0,0 +1,48 @@
+***************
+Making sketches
+***************
+
+
+
+Generating a schematic
+======================
+
+Consider a 1-dimensional scattering system, with scattering points located at
+angles ``theta`` w.r.t. the direct line of light (in the direction parallel to
```

Comment:
Really parallel to the effective proper motion.

Reply:
Noted